charts/gxf-platform/templates/config.yaml

```diff
@@ -56,10 +56,9 @@ data:
     response.url.notification.address=osgp-cucumber-tests-platform-common
 
     server=httpd
-    domain=:443
+    device.networkaddress=osgp-cucumber-tests-platform-publiclighting
```

Comment:
Does this still work in combination with common tests?

Reply:
yes